akka-stream/src/main/scala/akka/stream/impl/fusing/ActorGraphInterpreter.scala

```diff
@@ -269,12 +269,12 @@ private[stream] object ActorGraphInterpreter {
       }
 
     def exposedPublisher(publisher: ActorPublisher[Any]): Unit = {
+      exposedPublisher = publisher
```

Comment:
Why? There are thousands of such lines where you can check the validity of a previous value (null here) and if I add an assert here, then why not add assertions there? I.e. where do you draw the line?